docs/src/static-pages/shadows.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Shadows
+subtitle: Shadows are our richest visual identity element, enabling us to express an abstract sense of place
+category: guidelines
+---
+
+Shadows isolate a location's often overlooked beautiful details, and build on our visual identity's unique relationship with light.
+
+From nature to architecture, and abstract to prop, our shadow categories capture a wide range of details, allowing us to evoke straightforward or complex experiences.
+
+## Categories
+
+Our use of shadows falls into four categories, which are displayed below in pairs.
+
+### Nature and architecture
```

Comment:
I know it's a bit weird to pair them but the images are in pairs, so I thought I'll just make it work for now and maybe we'll fix it or get a contribution to fix it later. I'm 99% sure this page is never seen anyway.

Reply:
Seems fair for now - if someone updates the content to be more useful we can address layout then 👍